addresses/mainnet.json

```diff
@@ -1482,5 +1482,23 @@
     ],
     "status": "ACTIVE",
     "version": "v3"
+  },
+  "20250121-v3-stable-surge": {
+    "contracts": [
+      {
+        "name": "StableSurgePoolFactory",
+        "address": "0xD53F5d8d926fb2a0f7Be614B16e649B8aC102D83"
+      },
+      {
+        "name": "StableSurgeHook",
+        "address": "0xb18fA0cb5DE8cecB8899AAE6e38b1B7ed77885dA"
+      },
+      {
+        "name": "MockStablePool",
+        "address": "0xc05ff9851c2a000F3C319D2986D8712317583B79"
```

Comment:
Should this be mock?

Reply:
it's just the name :) the contract is the actual pool.

This is just a convention; we deploy a "mock pool" just to verify the contract during deployment.